### ISSUE TITLE
Update installation instructions for Nodejs/npm

### ISSUE THREAD

**Issue by diabolicallyrandom** [hexparrot/mineos-node#242]
The NPM packages in Debian are outdated, deprecated and have been removed from debian stretch (testing, releases in a couple weeks).

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=857986

Install instructions here keep the packages more properly up to date by obtaining them directly from nodesource.

https://nodejs.org/en/download/package-manager/

`curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - `
`sudo apt-get install -y nodejs`

**Comment by hexparrot** [comment]
Thanks so much for the heads-up.  Actually, according to the [MineOS wiki](https://minecraft.codeemo.com/mineoswiki/index.php?title=MineOS-node_(apt-get)) and all my [gists](https://gist.github.com/hexparrot), nodesource is definitely used.

I suppose I have just entirely glossed over that the README here on the repo itself was introducing wayyyy outdated information!

I have updated the README to reflect this and to encourage going to the wiki for greater detail in the installation process.

**Comment by diabolicallyrandom** [comment]
Just FYI, I was able to install and it is running just fine on Debian Stretch. I am using Node 6.x and OpenJDK 8.x.

Sidenote: The only issue I had was importing my servers from my old server. I copied the folders in as root, but logged into the webui using an actual user account - couldn't get the servers to show up, but basically I just had to correct the user and group ownership of the files and folders with a chown.
